src/lib/components/compositions/checkbox/animated.story.svelte

```diff
@@ -38,11 +38,11 @@
 </script>
 
 {#if withForm}
-  <Form onSubmitWhenValid={submitForm}>
+  <Form ariaDescribedBy="animatedCheckboxStory" onSubmitWhenValid={submitForm}>
```

Comment:
This new `ariaDescribedBy` prop is new, and I've added it, so we can link the headers, errors or messages to the form itself for better accessibility.